backend/routes.js

```diff
@@ -74,7 +74,24 @@ route.GET('/eventsBefore/{before}/{limit}', {}, async function (request, h) {
     if (!limit) return Boom.badRequest('missing limit')
     if (isNaN(parseInt(limit)) || parseInt(limit) <= 0) return Boom.badRequest('invalid limit')
 
-    const stream = await sbp('backend/db/streamEntriesBefore', before, limit)
+    const stream = await sbp('backend/db/streamEntriesBefore', before, parseInt(limit))
+    request.events.once('disconnect', stream.destroy.bind(stream))
+    return stream
+  } catch (err) {
+    return logger(err)
+  }
+})
+
+route.GET('/eventsBetween/{startHash}/{endHash}', {}, async function (request, h) {
+  try {
+    const { startHash, endHash } = request.params
+    const { offset } = request.query
+
+    if (!startHash) return Boom.badRequest('missing startHash')
+    if (!endHash) return Boom.badRequest('missing endHash')
+    if (isNaN(parseInt(offset)) || parseInt(offset) < 0) return Boom.badRequest('invalid offset')
```

Comment:
It should be fine to not specify the `offset` as it's an optional parameter.

Reply:
Nice catch.